### ISSUE TITLE
Pictures on nodes

### ISSUE THREAD

**Issue by gaborlaszlo** [makyo/polycul.es#44]
Would it be possible to attach pictures to the nodes, to be shown in the graph instead of the standard circle? It would require a file upload dialog, probably with a size limit. Also saved on export as inline encoded binary blobs or smthg?

**Comment by TangentFoxy** [comment]
> require a file upload dialog, probably with a size limit

Not necessarily. This is an idea I just had as well, and might attempt submitting a pull request for.
